src/rst.pest

```diff
@@ -285,7 +355,7 @@ marker = _{ (bullet_marker | "..") ~ " " }
 // star_line = { "****" ~ "*"* | spacechar ~ "*"+ ~ &spacechar }
 // ul_line = { "____" ~ "_"* | spacechar ~ "_"+ ~ &spacechar }
 
-// whitespace = { spacechar | NEWLINE }
+whitespace = { spacechar | NEWLINE }
```

Comment:
Neither whitespace nor spacechar later are used in other rules, and I hope they don’t need to be. Please leave them commented.

```suggestion
// whitespace = { spacechar | NEWLINE }
```

Reply:
My bad, I tried using the whitespace rule to deal with the space between the reference text and the bracketed target, before realizing that `` `<target_only>`_`` is a valid reference, so I just trimmed the space in the conversion.